nemo_skills/evaluation/metrics.py

```diff
@@ -164,10 +164,10 @@ def update(self, predictions, aggregation_mode):
             self.no_answer += predictions[0]['predicted_answer'] is None
         elif aggregation_mode == "reward":
             valid_answers_and_results = [
-                    (elem['predicted_answer'], elem['is_correct'], elem['score'])
-                    for elem in predictions
-                    if elem['predicted_answer'] is not None
-                ]
+                (elem['predicted_answer'], elem['is_correct'], elem['score'])
```

Comment:
The 'score' comes from Shubham's branch, but the current main has 'reward_model_score' as the default parameter name. Probably best to use that, since it's also more explicit. After we integrate everything into main, it will use the same script, so the naming would be unified

Reply:
ok, I can revise the naming